templates/web/base/report/_inspect.html

```diff
@@ -49,6 +49,11 @@ <h2 class="inspect-form-heading">[% loc('Inspect report') %]</h2>
         <p>
           <a href="#" class="btn btn--block btn--geolocate">[% loc('Set to my current location') %]</a>
         </p>
+        [% IF permissions.report_reject %]
+          <p>
+            <a href="[% c.uri_for( '/contact', { id => problem.id, reject => 1 } ) %]" class="btn btn--block">[% loc('Reject report') %]</a>
+          </p>
+        [% END %]
```

Comment:
I'm not sure I understand this functionality. The inspector clicks on reject which takes them to the contact form and then an email will be sent to the council contact address to say the report has been rejected? Why not just close the report?

Reply:
It's a workflow that BANES asked for specifically. The person receiving the rejection email is responsible for closing the report, not the person who rejects it, IYSWIM.